### ISSUE TITLE
cannot use verbose with migrate

### ISSUE THREAD

**Issue by cornernote** [joeyates/imap-backup#149]
Getting an error when trying to use `-v` with `migrate`:

```sh
imap-backup migrate SOURCE DEST -v
```

```
/var/lib/gems/3.0.0/gems/imap-backup-9.0.2/lib/imap/backup/cli/migrate.rb:18:in `initialize': unknown keyword: :verbose (ArgumentError)
	from /var/lib/gems/3.0.0/gems/imap-backup-9.0.2/lib/imap/backup/cli.rb:115:in `new'
	from /var/lib/gems/3.0.0/gems/imap-backup-9.0.2/lib/imap/backup/cli.rb:115:in `migrate'
	from /var/lib/gems/3.0.0/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
	from /var/lib/gems/3.0.0/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
	from /var/lib/gems/3.0.0/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
	from /var/lib/gems/3.0.0/gems/thor-1.2.1/lib/thor/base.rb:485:in `start'
	from /var/lib/gems/3.0.0/gems/imap-backup-9.0.2/bin/imap-backup:12:in `block in <top (required)>'
	from /var/lib/gems/3.0.0/gems/imap-backup-9.0.2/lib/imap/backup/logger.rb:34:in `sanitize_stderr'
	from /var/lib/gems/3.0.0/gems/imap-backup-9.0.2/bin/imap-backup:11:in `<top (required)>'
	from /usr/local/bin/imap-backup:25:in `load'
	from /usr/local/bin/imap-backup:25:in `<main>'
```

**Comment by joeyates** [comment]
Hi @cornernote

Thanks for notifying this problem. I'll let you know when it's fixed!

**Comment by joeyates** [comment]
I've released a fix for this in [#v9.1.1](https://github.com/joeyates/imap-backup/releases/tag/v9.1.1)
